ee/api/test/test_organization.py

```diff
@@ -59,7 +60,15 @@ def test_delete_second_managed_organization(self):
         self.assertFalse(Organization.objects.filter(id=organization.id).exists())
         self.assertFalse(Team.objects.filter(id=team.id).exists())
 
-    def test_delete_last_organization(self):
+        mock_capture.assert_called_once_with(
+            self.user.distinct_id,
+            "organization deleted",
+            ANY,
```

Comment:
Any reason why we don't assert the full payload? We can just get the ID from the `user.organization` object.

Reply:
Annoyingly it adds a few more lines of code to the test and I thought since the method specifically wasn't under test...

Added it back though.